salt/cloud/__init__.py

```diff
@@ -76,6 +76,12 @@ def _call(queue, args, kwargs):
             queue.put('ERROR')
             queue.put('Exception')
             queue.put('{0}\n{1}\n'.format(ex, trace))
+        except SystemExit as ex:
+            print("Communicatior: caught exception")
```

Comment:
Should this be a print or a log?

Reply:
That was a left-over debug print, sorry about that. Removed.